diff --git a/config/gazebo-iris-gimbal.parm b/config/gazebo-iris-gimbal.parm
index dd36c19..c14cdf3 100644
--- a/config/gazebo-iris-gimbal.parm
+++ b/config/gazebo-iris-gimbal.parm
@@ -30,3 +30,29 @@ RC8_OPTION       214         # Mount1 Yaw
 SERVO9_FUNCTION  8           # Mount1Roll
 SERVO10_FUNCTION 7           # Mount1Pitch
 SERVO11_FUNCTION 6           # Mount1Yaw
+
+# WP speed
+#WPNAV_SPEED 20
+#CIRCLE_OPTIONS 12
+
+# === MAVLink Gimbal v2 프로토콜 사용 ===
+#MNT1_TYPE 3          # 2 MAVLink Gimbal v2
+MNT1_DEFLT_MODE 0    # Default mode: Retract or neutral
+MNT1_NEUTRAL_X 0     # Neutral roll
+MNT1_NEUTRAL_Y 0     # Neutral pitch
+MNT1_NEUTRAL_Z 0     # Neutral yaw
+
+# === 짐벌 리드율 및 RC 제어 비활성화 ===
+MNT1_LEAD_PTCH 0     # Pitch lead compensation 비활성화
+MNT1_LEAD_RLL 0      # Roll lead compensation 비활성화
+MNT1_RC_RATE 0       # RC 명령 반영 비율 0
+
+# === RC 입력 비활성화 ===
+#RC6_OPTION 0         # RC6 기능 없음
+#RC7_OPTION 0         # RC7 기능 없음
+#RC8_OPTION 0         # RC8 기능 없음
+
+# === Servo 출력 비활성화 (MAVLink 제어만 허용) ===
+# SERVO9_FUNCTION 0    # Mount1 Roll
+#SERVO10_FUNCTION 0   # Mount1 Pitch
+#SERVO11_FUNCTION 0   # Mount1 Yaw
\ No newline at end of file
diff --git a/models/gimbal_small_3d/model.sdf b/models/gimbal_small_3d/model.sdf
index 0c4b014..ec0c475 100644
--- a/models/gimbal_small_3d/model.sdf
+++ b/models/gimbal_small_3d/model.sdf
@@ -191,10 +191,13 @@
       <sensor name="camera" type="camera">
         <pose>0 0 0 -1.57 -1.57 0</pose>
         <camera>
-          <horizontal_fov>2.0</horizontal_fov>
+          <!-- <horizontal_fov>2.0</horizontal_fov> -->
+          <horizontal_fov>1.4147</horizontal_fov>  <!-- rad(81°) ≃ 1.4147 -->
           <image>
-            <width>640</width>
-            <height>480</height>
+            <!-- <width>640</width>
+            <height>480</height> -->
+            <width>1920</width>   
+            <height>1080</height>  
           </image>
           <clip>
             <near>0.05</near>
