diff --git a/sdformat_urdf/CMakeLists.txt b/sdformat_urdf/CMakeLists.txt
index 91b0da5..38af6ba 100644
--- a/sdformat_urdf/CMakeLists.txt
+++ b/sdformat_urdf/CMakeLists.txt
@@ -14,7 +14,7 @@ find_package(ament_cmake_ros REQUIRED)
 
 find_package(pluginlib REQUIRED)
 find_package(rcutils REQUIRED)
-find_package(urdfdom_headers 1.0.6 REQUIRED)
+find_package(urdfdom_headers REQUIRED)
 find_package(urdf_parser_plugin REQUIRED)
 find_package(tinyxml2_vendor REQUIRED)
 find_package(TinyXML2 REQUIRED)
